apps/user-service/src/auth/dto/reset-password-response.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { JsonApiAttributes } from "@terramatch-microservices/common/dto/json-api-attributes";
+
+@JsonApiDto({ type: 'logins', id: 'number' })
```

Comment:
This is not a login. Please choose a different resource type. It should likely match the endpoints. In this case, `passwordResets` would make sense to me.